wallet-new/bench/Client/Cardano/Wallet/Web/Endpoint/GetHistory.hs

```diff
@@ -0,0 +1,50 @@
+-- | Function for running a client, for @GetHistory@.
+
+module Client.Cardano.Wallet.Web.Endpoint.GetHistory
+    ( getHistoryIO
+    ) where
+
+import           Universum
+
+import           Client.Cardano.Wallet.Web.Api     (getHistory)
+import           Client.Cardano.Wallet.Web.Run     (runEndpointClient)
+import           Client.Cardano.Wallet.Web.Analyze (analyzeResponseIfNeeded, checkResponse)
+import           Bench.Cardano.Wallet.Types        (BenchEndpoint (..), CompleteConfig (..),
+                                                    Wallet (..), WalletAccount (..),
+                                                    WalletsConfig (..), Response,
+                                                    ResponseReport (..))
+import           Bench.Cardano.Wallet.Random       (pickRandomElementFrom)
+
+import           Pos.Wallet.Web.ClientTypes        (Addr, CHash (..), CId (..), CTx (..))
+
+-- | Run 'GetHistory' client. As a result we will get
+-- a list of transactions and size of a full history.
+getHistoryIO :: CompleteConfig -> IO ()
```

Comment:
Generally, it seems that there is an interface hidden in all these client calls. Not saying it's necessary, just that it's there.
```
class ClientCall ... where
    callEndpoint :: CompleteConfig -> IO ()  
    analyze :: Needs to be parametrized
```
These individual modules seem kind of... verbose? Not really sure, but something feels off :man_shrugging: